TritonSE/Supply-Chain
# Feature/hahsu/921130332 boilerplate

### Administrative Info
Monday Board ID: 853628905
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
Created the boilerplate code: express backend, react frontend. 

### Testing
How did you confirm your changes worked? 
1. cd into backend folder
2. run the command "npm start"
3. cd into client folder
4. run the command "npm start"
5. Go to browser: http://localhost:3000/ and you should see "Supply Chain" centered.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.
<img width="1437" alt="Screen Shot 2020-12-19 at 1 33 25 PM" src="https://user-images.githubusercontent.com/26681171/102700067-eb4b2080-41fe-11eb-86ae-2820e9a69000.png">
